types/decentlab/IAM/default.schema.json

```diff
@@ -3,67 +3,68 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Default",
   "type": "object",
-  "allOf": [
```

Comment:
why this change?

Reply:
You cant create a rule with the decentlab IAM. I figured it cant load the schema so i'am removing it.

Reply:
ok get it, this is something we will have to support to allow this schema inheritance